app/about/page.tsx

```diff
@@ -6,86 +6,146 @@ const DEFAULT_IMAGE = "/icons/app/stellarHorizons.png" as const;
 const team = [
   {
     name: "Mauricio Curiel",
-    image: DEFAULT_IMAGE,
+    image: "/images/team/mau.png",
     description:
       "Project Manager overseeing development and project milestones",
+    email: "",
   },
   {
     name: "Ahsan Ali",
-    image: DEFAULT_IMAGE,
+    image: "/images/team/Ahsan.png",
     description:
       "Backend Developer specializing in server-side development and testing.",
+    email: "",
   },
   {
     name: "Ali Altimimi",
-    image: DEFAULT_IMAGE,
+    image: "/images/team/Ali.png",
     description:
       "Backend Developer focused on database architecture and optimization.",
+    email: "",
   },
   {
     name: "Connor Linville",
-    image: DEFAULT_IMAGE,
+    image: "/images/team/Connor.png",
     description:
       "Backend Developer with expertise in deployment strategies and user authentication systems.",
+    email: "",
   },
   {
     name: "Steven Lu",
-    image: DEFAULT_IMAGE,
+    image: "/images/team/Steven.png",
     description:
       "Frontend Developer responsible for implementing designs from Figma into a responsive UI.",
+    email: "",
   },
   {
     name: "Raymund Mercader",
-    image: DEFAULT_IMAGE,
+    image: "/images/team/Raymund.png",
     description:
       "Full Stack Developer experienced in both backend and frontend development.",
+    email: "",
   },
   {
-    name: "Minh Nguyen",
-    image: DEFAULT_IMAGE,
+    name: "Hoa Tuong Minh Nguyen",
+    image: "/images/team/Minh.png",
     description:
-      "UI/UX Designer skilled in creating intuitive user interfaces using Figma.",
+      "UI/UX Designer on Figma and Frontend Developer.",
+    email: "hoatuongminh@gmail.com",
   },
   {
     name: "Ryan Tang",
-    image: DEFAULT_IMAGE,
-    description: "Backend Developer with expertise in payment systems.",
+    image: "/images/team/Ryan.png",
+    description: "Backend Developer focused on stripe and prisma integration.",
+    email: "",
   },
 ];
 
 const AboutPage = () => {
   return (
     <div className="min-h-screen bg-gray-100 text-gray-800 p-8">
-      {/* Team Introduction */}
-      <section className="text-center mb-12">
-        <h1 className="text-5xl font-bold mb-6">Meet the LikeHome Team</h1>
-        <p className="text-lg">
-          We are a group of passionate developers committed to building the best
-          hotel-booking platform.
-        </p>
-      </section>
+      {/* Main Flex Container */}
+      <div className="flex flex-col lg:flex-row gap-8">
```

Comment:
```suggestion
      <div className="flex flex-col gap-8">
```
Contact us is typically at the bottom. Also this layout has a lot of dead space.